app/models/project/phase.rb

```diff
@@ -85,25 +85,30 @@ def calculate_duration
     Day.working.from_range(from: start_date, to: finish_date).count
   end
 
-  def follows_previous_phase?
-    !!previous_phase&.date_range_set?
-  end
-
   def default_start_date
     return @default_start_date if defined?(@default_start_date)
 
     previous_finish_date = previous_phase&.finish_date
     @default_start_date = previous_finish_date && Day.next_working(from: previous_finish_date).date
   end
 
-  private
+  def previous_phases
+    return @previous_phases if defined?(@previous_phases)
+
+    @previous_phases = project.available_phases.select { it.position < position }
+  end
 
   def previous_phase
-    return @previous_phase if defined?(@previous_phase)
+    previous_phases.last
+  end
+
+  def follows_previous_phase?
+    !!previous_phase&.date_range_set?
+  end
+
+  def following_phases
+    return @following_phases if defined?(@following_phases)
 
-    @previous_phase = project
-     .available_phases
-     .select { it.position < position }
-     .last
+    @following_phases = project.available_phases.select { it.position > position }
```

Comment:
Same here (UI didn't allow me to select all lines)
```suggestion
    @following_phases ||= project.available_phases.select { it.position > position }
```